README.md

```diff
@@ -178,6 +194,8 @@ repository (i.e. [general issues in Zonemaster](https://github.com/zonemaster/zo
 
 ## Notable bugs and issues
 
+*TO BE UPDATED AT RELEASE*
+
```

Comment:
There's a step for this in the release process. I don't think we need to spell it out in the document.

Reply:
I just wanted to have something sticking in the eyes when looking at the document in the develop branch.